openslides/users/management/commands/createopenslidesuser.py

```diff
@@ -26,10 +26,16 @@ def add_arguments(self, parser):
             'password',
             help='The password of the new user.'
         )
+        parser.add_argument(
+            'groups_id',
+            help='The group ids of the new user.'
+        )
 
     def handle(self, *args, **options):
         user_data = {
             'first_name': options['first_name'],
             'last_name': options['last_name'],
         }
-        User.objects.create_user(options['username'], options['password'], **user_data)
+        user = User.objects.create_user(options['username'], options['password'], **user_data)
+        user.save()
+        user.groups.add(Group.objects.get(id=options['groups_id']))
```

Comment:
Also you need to validate the input. Only a string that can be transformed to an integer is allowed.